src/main/java/svc/controllers/CourtController.java

```diff
@@ -30,11 +34,23 @@ CourtsDTO GetCourts() {
 	
 	@ResponseBody
 	@RequestMapping(method = RequestMethod.GET, value="/{id}")
-	Court GetCourt(@PathVariable("id") Long id) {
-		if (id == null) {
-			LogSystem.LogEvent("Null id passed to controller");
-		}
-		
-		return courtManager.GetCourtById(id);
+	Court GetCourt(@PathVariable("id") Long id) throws NotFoundException {
+		Court court = courtManager.GetCourtById(id);
+		if (court == null) 
+			throw new NotFoundException();

```

Comment:
Please prove an error message here in the exception. So note below about GlobalExceptionHandler....